airflow-core/tests/unit/api_fastapi/core_api/routes/public/test_connections.py

```diff
@@ -390,7 +401,13 @@ def test_should_respond_403(self, unauthorized_test_client):
         "body, updated_connection, update_mask",
         [
             (
-                {"connection_id": TEST_CONN_ID, "conn_type": TEST_CONN_TYPE, "extra": '{"key": "var"}'},
+                {
+                    "connection_id": TEST_CONN_ID,
+                    "conn_type": TEST_CONN_TYPE,
+                    "extra": '{"key": "var"}',
+                    "login": TEST_CONN_LOGIN,
+                    "port": TEST_CONN_PORT,
+                },
```

Comment:
It seems that with this change we cannot explicitely set to `None`. (insert a None value on purpose)

Reply:
Okay, I assume not it is considered- as discussed in 1:1 slack - that body fields are only set as provided when no update mask provided.

Extended pytest on this to catch a regression as well, was not fully covered in tests before.

Reply:
Nice thanks for adding that test. Indeed a test case was missing